react/features/connection-indicator/components/web/ConnectionIndicatorContent.js

```diff
@@ -323,40 +320,36 @@ export function _mapStateToProps(state: Object, ownProps: Props) {
     const conference = state['features/base/conference'].conference;
     const participant
         = participantId ? getParticipantById(state, participantId) : getLocalParticipant(state);
-    const firstVideoTrack = getTrackByMediaTypeAndParticipant(
-        state['features/base/tracks'], MEDIA_TYPE.VIDEO, participantId);
     const sourceNameSignalingEnabled = getSourceNameSignalingFeatureFlag(state);
 
+    const tracks = state['features/base/tracks'];
+    const audioTrack = getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.AUDIO, participantId);
+    let videoTrack = getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.VIDEO, participantId);
```

Comment:
You can use [getVideoTrackByParticipant](https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/tracks/functions.ts#L431) here instead.

Reply:
I just moved this code around, and will consider it if I further refactor things!